src/pretix/static/pretixbase/js/asynctask.js

```diff
@@ -218,10 +218,16 @@ $(function () {
             'this page and try again.'
         ));
 
-
-        console.log($(this).get(0))
         var formData = new FormData($(this).get(0))
         formData.append('ajax', '1');
+        if (e.originalEvent.submitter) {
+            // Not supported on IE, may lead to wrong results, but we don't support IE in the backend anymore
+            console.warn('Unknown submitter, IE unsupported');
```

Comment:
We do not need an IE fallback, since
- we've dropped caring about IE in the backend a year a go, we show a big fat warning to IE users since May 2021 and nobody complained
- we still support IE in the frontend, but we have no data-asynctask with ``<button name=…>`` in the current frontewnd
- IE11 is officially retired by Microsoft *next Wednesday*. We haven't formally decided or announced this yet, but I think we can also start stop caring about it in the frontend very very soon, and we're unlikely to introduce a new button in the frontend by then ;)

However, we *should* care on Safari. B ut as I understand you

Reply:
Yes, we are fine with older Safari-versions. Even current Safari (15.5) shows the same behaviour regarding buttons name-value-pair being added to the FormData.